tools/fv_io.F90

```diff
@@ -789,7 +832,7 @@ subroutine  fv_io_write_restart(Atm, timestamp)
       fname = 'RESTART/fv_core.res'//trim(suffix)//'.nc'
     endif
 
-    Atm%Fv_restart_tile_is_open = open_file(Atm%Fv_restart_tile, fname, "overwrite", fv_domain, is_restart=.true.)
+    Atm%Fv_restart_tile_is_open = open_file(Atm%Fv_restart_tile, fname, "overwrite", fv_domain, nc_format="netcdf4", is_restart=.true.)
```

Comment:
Instead of hard codding `netcdf4` here, you can set this namelist
https://github.com/NOAA-GFDL/FMS/blob/d8208e61be2dff46cc387cbace0f1f7c2bd950fc/fms2_io/fms2_io.F90#L382-L384

Reply:
What will happen if user forgets to set that namelist parameter, for whatever reason, will FMS just ignore chunksizes argument or report an error and terminate the program?

Reply:
It will output a warning and ignore them.
https://github.com/NOAA-GFDL/FMS/blob/main/fms2_io/netcdf_io.F90#L951

Reply:
In netcdf_io.F90, here:

https://github.com/NOAA-GFDL/FMS/blob/073c6003b90099c3b02084b087601ff3109e9f83/fms2_io/netcdf_io.F90#L621

`fileobj%is_netcdf4` is set to `.true.` only based on `nc_format` function argument, not based on namelist parameter ater line [629](https://github.com/NOAA-GFDL/FMS/blob/073c6003b90099c3b02084b087601ff3109e9f83/fms2_io/netcdf_io.F90#L629).

Reply:
ahh this is a bug, thanks for letting us know. We will get this fixed!